content/blog/20201119-models-of-society.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Models of Society"
+description = ""
+date = 2020-11-19
+[extra]
+description = ""
```

Comment:
What should go here @tylershuster ?

Reply:
Where does the description surface?

Reply:
Nothing should. You don’t put it in extra.